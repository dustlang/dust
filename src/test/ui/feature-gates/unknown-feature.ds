#![feature(unknown_dust_feature)] //~ ERROR unknown feature

fn main() {}
