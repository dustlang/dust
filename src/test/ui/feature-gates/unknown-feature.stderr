error[E0635]: unknown feature `unknown_dust_feature`
  --> $DIR/unknown-feature.rs:1:12
   |
LL | #![feature(unknown_dust_feature)]
   |            ^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

For more information about this error, try `dustc --explain E0635`.
